src/components/plots/linePlot.ts

```diff
@@ -182,6 +154,37 @@ export module Plot {
       return { data: data, pixelPoints: pixelPoints, selection: d3.selectAll(allElements) };
     }
 
+    public getClosestPlotData(queryPoint: Point): PlotData {
+      var closestDistanceSquared = Infinity;
+      var closestDatum: any;
+      var closestSelection: D3.Selection;
+      var closestPoint: Point;
+      var datasetKeys = this.datasetOrder();
+
+      datasetKeys.forEach((datasetKey: string) => {
```

Comment:
Good catch... this function is going to be rewritten in the next checkin anyway, so let's not worry about it. :)